### ISSUE TITLE
Plugins from gulp blacklist

### ISSUE THREAD

**Issue by maxmalov** [Pestov/essential-gulp-plugins#2]
"Official" blacklist: https://github.com/gulpjs/plugins/blob/master/src/blackList.json

For example, `gulp-requirejs` and `gulp-browserify` are blacklisted plugins, since you can use `rjs` and `browserify` directly in gulp tasks.

**Comment by Pestov** [comment]
Yes, you are right. So what you think about how I can mark this task? I must remove them or select them as [offical recipes by gulp](https://github.com/gulpjs/gulp/tree/master/docs/recipes) team in new section?

**Comment by maxmalov** [comment]
I think the additional section with recipes would be perfect. Besides you can mention there that not all cases require a specific gulp plugin (difference from grunt), because you can use corresponding tools directly. Also additional notification about blacklisted plugins can be added. Smth like:

### Recipes

[browserify](https://github.com/gulpjs/gulp/blob/master/docs/recipes/browserify-uglify-sourcemap.md) - optimize the work with and Browserify. _Do not use [gulp-browserify](https://github.com/gulpjs/plugins/blob/master/src/blackList.json#L8)_

What do you think about that?

**Comment by Pestov** [comment]
Sounds good. Thanks. I will add it in the near future.
